samples/Authorization/audit-event-hub-autorization/README.md

```diff
@@ -0,0 +1,26 @@
+# Audit authorization rules on Event Hub namespaces
+
+Event Hub clients should not use a namespace level access policy that provides access to all queues and topics in a namespace. To align with the least previlege security model, you shoud create access policies at the entity level for queues and topics to provide access to only the specific entity
```

Comment:
typo here and in policy description: least **privilege**

(I know this isn't your change...I'll make a note to fix this is in the built-in too)